scipy/signal/_filter_design.py

```diff
@@ -4379,12 +4379,12 @@ def cheb1ap(N, rp):
 
 def cheb2ap(N, rs):
     """
-    Return (z,p,k) for Nth-order Chebyshev type I analog lowpass filter.
+    Return (z,p,k) for Nth-order Chebyshev type II analog lowpass filter.
 
-    The returned filter prototype has `rs` decibels of ripple in the stopband.
+    The returned filter prototype has attenuation of at least ``rs`` decibels in the stopband.
 
     The filter's angular (e.g. rad/s) cutoff frequency is normalized to 1,
-    defined as the point at which the gain first reaches ``-rs``.
+    defined as the point at which the gain first drops below ``-rs``.
```

Comment:
Looks good to me.

A nitpick: [Wikipedia](https://en.wikipedia.org/wiki/Chebyshev_filter#Type_II_Chebyshev_filters_(inverse_Chebyshev_filters)) phrases this as 
> the smallest frequency at which this maximum is attained
where "maximum" means "maximum gain in stop-band" (wording probably unclear out of context).

How about
```suggestion
    defined as the point at which the attenuation first reaches ``rs``.
```

Now we only use "attenuation" in the docstring, so the reader doesn't have to mentally switch from that to "gain".

Reply:
Looks good. As long as it stays factually correct, it shouldn't matter much, but I like the idea of the reader staying in the same mindset.